src/System.Windows.Forms/src/System/Windows/Forms/IToolTip.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Windows.Forms
+{
+    internal interface IToolTip
+    {
+        /// <summary>
+        ///  Returns AccessibilityObject of an element (Control, DataGridViewElement, ToolStripItem, etc.).
+        /// </summary>
+        /// <remarks>
+        ///  Return null if an element doesn't have AccessibilityObject.
+        /// </remarks>
+        AccessibleObject GetAccessibilityObjectForToolTip();
```

Comment:
Is there any reason we can't make it a property? Just looks nicer at callsites
```suggestion
        AccessibleObject AccessibilityObjectForToolTip { get; }
```

Reply:
No reasons. I'll change it, thanks.

Reply:
Changed the method to the property.